public/js/index.js

```diff
@@ -524,6 +524,62 @@ Index.loadContextMenuCategories = (catList, id) => Server.callAPI(`/api/archives
     },
 );
 
+/**
+ * Build rating options for contextMenu and select the one for the current ID.
+ * @param {*} id The ID of the archive to check
+ * @returns Ratings
+ */
+Index.loadContextMenuRatings = (id) => Server.callAPI(`/api/archives/${id}/metadata`, "GET", null, `Error finding metadata for ${id}!`,
```

Comment:
I'm not a massive fan of making a new API request to get metadata again when it's already technically present in the page.. But it'd be a bit annoying to hook into the datatables part to copy the tags somewhere so it's fine for now.

Reply:
Yep, tried to get the data but it was quite messy.